### DISCUSSION TITLE
Error | itwin-connector.Framework | Lock(s) is owned by another Briefcase

### DISCUSSION THREAD

**Discussion by PeterJBassett** [iTwin/community#47] [Q&A] [answered]
Hello 
In my connector application syncing FutureOn projects to Plantsight projects intermittent locking errors are being reported.
For example:
- sync project - successful
- re-sync project -> Error | itwin-connector.Framework | Lock(s) is owned by another Briefcase
- re-sync project again - successful

Is there a possible reason why intermittent locking errors might occur?

**Comment by pmconne** [comment]
@jffmarker @abeesh

**Comment by abeesh** [comment, marked as answer]
It is possible that another application, besides your connector, may be temporarily locking the iModel to push some shared information. Please check the following:

1. Are there any other synchronization jobs running at the same time?
2. Use the Lock REST API to check who holds the lock. https://developer.bentley.com/apis/imodels-v2/operations/get-imodel-locks/"

**Reply by PeterJBassett** [reply]
Thanks @abeesh I guess it's possible that some other application was accessing the iModel at the same time as the sync. I don't have visibility of other applications that might be able to access the iModel.
